src/test/java/guitests/ContextMenuTests.java

```diff
@@ -81,6 +80,16 @@ public void test2() {
     }
 
     /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_tenIssuesInListView_markIssue9andBelowReadUnread() {
+        bringSelectedAndIssuesBelowInFocus(7);
+        markIssuesBelow(7, true);
+        markIssuesBelow(7, false);
```

Comment:
It's not a matter of blindly following my comments. The current test is not strong enough. The case I mentioned is just an example. Try to figure out the corner cases and come up with a stronger set of test cases. 
BTW, are you testing the 'no loading when right-click' behavior?